cpp/open3d/geometry/VoxelGrid.h

```diff
@@ -189,30 +189,42 @@ class VoxelGrid : public Geometry3D {
                                                   double height,
                                                   double depth);
 
+    /// \enum VoxelColorMode
+    ///
+    /// \brief Possible ways of determining voxel color from PointCloud.
+    enum class VoxelColorMode { AVG, MIN, MAX, SUM };
```

Comment:
Possible nitpick: can we call this "VoxelPoolingMode" or "VoxelPoolingMethod" instead of "VoxelColorMode"?

Reply:
Thanks for the feedback! I'd tend to agree that "VoxelPoolingMode" is probably the better name, but I'll wait until a maintainer has had the chance to review the PR to get their feedback.

Reply:
@rxba Thanks for this feature contribution! It looks very useful.
@IDoCodingStuffs I agree, "VoxelPoolingMode" it is